src/DataFrame/IO/Parquet/Thrift.hs

```diff
@@ -313,7 +312,6 @@ readFileMetaData metadata metaDataBuf bufferPos lastFieldId fieldStack = do
                         then readVarIntFromBuffer @Int metaDataBuf bufferPos
                         else return $ fromIntegral ((sizeAndType `shiftR` 4) .&. 0x0f)
 
-                let elemType = toTType sizeAndType
```

Comment:
Similar comment for all these other elem types. Actually since there are so many of them it might be good to actually write out the errors.

So maybe comment them out then I'll follow up by adding them back  and explicitly throwing on mismatch. That way we can separate concerns.